webpack.config.dev.js

```diff
@@ -0,0 +1,22 @@
+const merge = require('webpack-merge');
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
+
+let dev = {
```

Comment:
`const` means that the binding can't change. It has also a semantical meaning. If a value is not going to change (like in this case) `const` just makes sense. You scan the code and realise out of the way, which values are going to change and which aren't even if the keys are mutable.

[This](https://ponyfoo.com/articles/var-let-const) is a good article on the subject which gets deeper on the meanings for the TDZ and constrains as better ways to write code.